### DISCUSSION TITLE
Language Server Crashed

### DISCUSSION THREAD

**Discussion by RoserFwk** [SPGoding/datapack-language-server#1458] [Q&A] [answered]
After updated to v4.0.0 the extension is just not working. It's similar to the situation mentioned in #1450 but do have difference.

Error infomation included:
![图片](https://github.com/SpyglassMC/Spyglass/assets/121746494/86687167-3160-4dca-b7fa-bc57a2953b55)
![图片](https://github.com/SpyglassMC/Spyglass/assets/121746494/202afbd9-26e8-4484-9274-05c311de0a05)
![图片](https://github.com/SpyglassMC/Spyglass/assets/121746494/ab552208-388e-4d8f-92b5-eb41977deffe)

Outputs:
[output.txt](https://github.com/user-attachments/files/16067284/output.txt)
[output(1).txt](https://github.com/user-attachments/files/16067285/output.1.txt)

Help needed,  is there any ideas?

**Comment by misode** [comment]
It failed to download necessary resources for the language server to function. Is `raw.githubusercontent.com` blocked in your country?

Try using another data source by creating a `spyglass.json` file at the root of your workspace:
```json
{
  "env": {
    "env": "jsDelivr"
  }
}
```

**Reply by RoserFwk** [reply]
Due to it is acutally my DNS problem, so another data source didn't fix that. But also thanks to your help.

**Comment by RoserFwk** [comment, marked as answer]
Problem have been solved by config my DNS, it seems that it is working as intended now.
